index.html

```diff
@@ -1,277 +1,334 @@
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section>
-    <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section>
-     <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section>
-    <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section>
-    <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section> 
-    <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section> 
+    <!-- headline-container end -->
+
+
+      <!-- My Recent Works section start  -->
+      <section class="work">
+        <div class="work-header">
+          
+          <h2 class="work-title">My Recent Works</h2>
+          <div class="work-header-img"></div>
+          <img src="images/work-title-img.svg" alt="">
+        </div>
```

Comment:
I have observed the this heading seems aligned on the right a little bit.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221350168-19f3e0cd-a167-4a6a-b90c-46ec939212e6.png)| ![image](https://user-images.githubusercontent.com/61361037/221350146-bc4fe74e-2feb-470f-8941-c66c2c365b3f.png)|